web/Global.asax

```diff
@@ -16,6 +16,16 @@
             System.Web.Helpers.WebMail.SmtpPort = (int)ConfigurationManager.AppSettings["smtpPort"].AsLong();
             System.Web.Helpers.WebMail.EnableSsl = ConfigurationManager.AppSettings["smtpEnableSsl"] == "true";
         }
+        
+        i18n.LocalizedApplication.Current.TweakMessageTranslation = delegate(System.Web.HttpContextBase context, i18n.Helpers.Nugget nugget, i18n.LanguageTag langtag, string message)
+        {
+            switch (context.Response.ContentType)
+            {
+                case "application/javascript":
+                    return message.Replace("\'", "\\'");
```

Comment:
Similar to the web.config comment, since we are not translating js files 'on the fly' with i18n module, this should not be needed, right?

BUT, something similar for JSON (since is what the /api calls returns), detecting "application/json" as MIME type, and scape double quotation marks (") since JSON doesn't allow single quotation marks to wrap strings. Or this comes built-in in the module?
Need to do some testing.

Reply:
you're right, some of the translations include double quotes which need to be handled. for example Type the name of the consultation, e.g. "{0}" requires at least one selection (in web\app_code\lcrest\userjobtitleserviceattributes.cs)
this needs to be escaped in the code, but not in the translation
I've added tests so that we can check in tests/testrest with /api/v1/en-US/Test?i18n=escape+this